src/renderer/entities/multisig/model/multisigs-model.ts

```diff
@@ -81,48 +81,56 @@ type GetMultisigsParams = {
   accounts: AnyAccount[];
 };
 
-const getMultisigsFx = createEffect(({ chains, accounts }: GetMultisigsParams): Promise<MultisigResult[]> => {
-  const requests = chains.flatMap((chain) => {
-    const multisigIndexer = networkUtils.getProxyExternalApi(chain);
+const getMultisigsFx = createEffect(({ chains, accounts }: GetMultisigsParams) => {
+  if (accounts.length === 0) return [];
 
-    if (nullable(multisigIndexer) || accounts.length === 0) return [];
+  const bindedRequestIdentities = scopeBind(requestIdentitiesFx, { safe: true });
+  const requests = chains.flatMap(async (chain) => {
+    const multisigIndexer = networkUtils.getProxyExternalApi(chain);
+    if (nullable(multisigIndexer)) return [];
 
     const client = new GraphQLClient(multisigIndexer.url);
-    const accountIds = uniq(
-      accounts.filter((a) => accountUtils.isChainIdMatch(a, chain.chainId)).map((account) => account.accountId),
-    );
+    const chainAccounts = accounts.filter((a) => accountUtils.isChainIdMatch(a, chain.chainId));
+    const accountIds = uniq(chainAccounts.map((account) => account.accountId));
 
     if (accountIds.length === 0) return [];
 
-    return multisigService.filterMultisigsAccounts(client, accountIds, chain);
+    const multisigs = await multisigService.filterMultisigsAccounts(client, accountIds, chain);
+    try {
+      // Identity request with drop after 5 seconds
+      const identities = await Promise.race([
+        bindedRequestIdentities({
+          accounts: multisigs.map((x) => x.accountId),
+          chainId: chain.chainId,
```

Comment:
Should we rather use identity chain? Or is it handled inside `requestIdentitiesFx`?

Reply:
requestIdentitiesFx tries to find identity chain under the hood, but it's not the case for most networks.